auth/handlers/oauth2.go

```diff
@@ -186,3 +192,47 @@ func getToken(res *http.Response) (GitHubAccessToken, error) {
 
 	return token, fmt.Errorf("no body received from server")
 }
+
+func getUserOrganizations(username, accessToken string) (string, error) {
+
+	organizations := []Organization{}
+	apiURL := fmt.Sprintf("https://api.github.com/users/%s/orgs", username)
+
+	req, reqErr := http.NewRequest(http.MethodGet, apiURL, nil)
+	if reqErr != nil {
+		return "", reqErr
+	}
+	req.Header.Add("Authorization", "token "+accessToken)
+
+	client := http.DefaultClient
+	resp, respErr := client.Do(req)
+	if respErr != nil {
+		return "", respErr
+	}
+	if resp.Body == nil {
+		return "", fmt.Errorf("empty response body")
```

Comment:
response from GitHub organizations: empty response body

Reply:
`if resp.Body != nil {`

Reply:
covered 2nd if and changed error as proposed ^